tier1/keyvalues3.cpp

```diff
@@ -1758,6 +1837,40 @@ void CKeyValues3Table::CopyFrom( KeyValues3 *parent, const CKeyValues3Table* src
 		EnableFastSearch();
 }
 
+void CKeyValues3Table::RenameMember( KeyValues3 *parent, KV3MemberId_t id, const CKV3MemberName &newName )
```

Comment:
While you at it, how about adding an assert to ensure the id is within valid range, for this method and any other that take id (like RemoveMember)